src/components/TabBarIcon.js

```diff
@@ -8,8 +9,12 @@ export default function TabBarIcon(props) {
     <Ionicons
       name={props.name}
       size={normalize(26)}
```

Comment:
Here I should use OrientationContext, when orientation=== 'landscape' then TabBarIcon should increase its size in order to looks like an svg icon which is slightly larger